Makefile

```diff
@@ -12,7 +15,11 @@ docs:
 test:
 	go test ./... -v
 
+thirdparty-licenses:
+	go install github.com/google/go-licenses
+	~/go/bin/go-licenses csv github.com/datadog/stratus-red-team/cmd/stratus | sort > $(ROOT_DIR)/LICENSE-3rdparty.csv	
```

Comment:
You're making an assumption that GOPATH is `~/go`

Reply:
Fixed to use $GOPATH